ERCS/erc-7715.md

```diff
@@ -165,13 +222,13 @@ If the request is malformed or the wallet is unable/unwilling to grant permissio
 }
 ```
 
-### Signer & Permission Types
+### Signer & Permission/Policy Types
 
-In this ERC, we specify a list of signers and permissions that we expect to be commonly used.
+In this ERC, we specify a list of signers, permissions and policies that we expect to be commonly used.
 
-This ERC does not specify an exhaustive list of signer or permission types, since we expect more signer/permission types to be developed as wallets get more advanced. A signer/permission type is valid as long as both the DApp and the wallet are willing to support it.
+This ERC does not specify an exhaustive list of signer or permission/policy types, since we expect more signer and permission/policy types to be developed as wallets get more advanced. A signer type and permission/policy type is valid as long as both the DApp and the wallet are willing to support it.
 
-However, if two signers or two permissions share the same type name, a DApp could request with one type of signer/permission while the wallet grants another. Therefore, it’s important that no two signers or two permissions share the same type. Therefore, new signer or permission types should be specified in ERCs, either in this ERC as an amendment or in another ERC.
+However, if two signers or two permissions/policies share the same type name, a DApp could request with one type of signer or permission/policy while the wallet grants another. Therefore, it’s important that no two signers or two permissions/policies share the same type. Therefore, new signer or permission/policy types should be specified in ERCs, either in this ERC as an amendment or in another ERC.
```

Comment:
I would add after this paragraph:

Local policies (policies defined under a specific permission) may have the same type with other local policies under a different permission, but may not have the same type with other local policies under the same permission.  Global policies should always have different type names.

Reply:
Agreed 👍

Reply:
Not sure I understand the "Local" vs "Global" policy distinction. Aren't global policies still defined under the Permission they apply to -- they're global in the sense that they can be used on any Permission, not that they always have a global scope?

Could it be simpler to understand to keep names globally unique, and then caveat
- Most policies apply only to certain Permissions
- Specifically labeled "Global" policies (e.g. rate-limit, gas-limit) can be applied to any Permission